src/components/Player.js

```diff
@@ -1,20 +1,37 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import "./index.css";
 import mode from "../assets/mode.png";
-import p1u from "../assets/p1u.png";
-import p2u from "../assets/p2u.png";
-import p1s from "../assets/p1s.png";
-import p2s from "../assets/p2s.png";
 import next from "../assets/next.png";
 import homeicon from "../assets/homeicon.png";
 import logo from "../assets/logo.png";
 import newselection from "../assets/audio/selectplayer.mp3";
 import { Link } from "react-router-dom";
 import Footer from "./Footer";
 import { interactCall } from "../services/callTelemetryIntract";
+import jwt from "jwt-decode";
 
 function Player() {
   const [current, setCurrent] = useState("");
+  
+  const [Player1,setPlayer1] = useState('');
```

Comment:
use common method to avoid the code duplication

Reply:
Created Another file added logic in that file and using it wherever it required